engine/swarm/swarm_manager_locking.md

```diff
@@ -0,0 +1,162 @@
+---
+description: Automatically lock Swarm managers to protect encryption keys
+keywords:
+- swarm, manager, lock, unlock, autolock, encryption
+title: Lock your swarm to protect its encryption key
+---
+
+In Docker 1.13 and higher, the RAFT logs used by swarm managers are encrypted on
+disk by default. This at-rest encryption protects your service's configuration
+and data from attackers who gain access to the encrypted RAFT logs. One of the
+reasons this feature was introduced was in support of the new [Docker
+secrets](secrets.md) feature.
+
+When Docker restarts, both the TLS key used to encrypt communication among swarm
+nodes, and the key used to encrypt and decrypt RAFT logs on disk, are loaded
+into each manager node's running memory from disk. Docker 1.13 introduces the
+ability to protect the mutual TLS encryption key and the key used to encrypt and
+decrypt RAFT logs at rest (on disk), by allowing you to take ownership of this
+key and to require manual unlocking of your managers. This feature is called
+_autolocking_ your swarm.
+
+When Docker restarts or whenever the encryption key needs to be loaded from disk
```

Comment:
"When a Docker restarts and needs to access the encrypted RAFT state on disk, you must..."

There aren't other motives to unlock a manager other than starting it (or restarting it).